scip.proto

```diff
@@ -326,18 +335,20 @@ message Occurrence {
   // (optional) The symbol that appears at this position. See
   // `SymbolInformation.symbol` for how to format symbols as strings.
   string symbol = 2;
-  // (optional) Bitmask for what `SymbolRole` apply to this occurrence. See
-  // `SymbolRole` for how to read and write this field.
+  // (optional) Bitset containing `SymbolRole`s in this occurrence.
+  // See `SymbolRole`'s documentation for how to read and write this field.
   int32 symbol_roles = 3;
-  // (optional) Markdown-formatted documentation for this specific range.  If
+  // TODO[Varun]: Specify dialect of Markdown.
```

Comment:
TODO for discussion/clarification.

Reply:
I'd go with CommonMark rather ran GH-flavored https://commonmark.org

It seems like Prettier targeted CommonMark when they added support to format Markdown files https://prettier.io/blog/2017/11/07/1.8.0.html

Reply:
Removed TODO and replaced Markdown with CommonMark.